feature: Create Helm charts for docx/pdf-converter images. Touches: cksource/cs#12191. (#15)
cksource/ckeditor-cs-on-premises-infrastructure: kubernetes/helm/ckeditor-pdf-converter/README.md

# CKEditor PDF Converter On-Premises Helm chart

Use this Helm chart to provision CKEditor PDF Converter on your
Kubernetes cluster.

> ## Minimum requirements
> - 1 CPU Core
> - 512MB RAM

Reviewer: The default setting provisions two instances of the converter

```suggestion
## Minimum requirements
- 2 CPU Cores
- 1024MB RAM
```

- Kubernetes 1.19+
- Helm v3

> The default configuration of running this service requires the reservation of 1
> CPU core and 512MB of RAM in the cluster. For more information about resources
> usage look here:

Reviewer: ```suggestion
The default configuration of running this service requires the reservation of 2
CPU cores and 1024MB of RAM in the cluster. For more information about resources
usage look here:
```

Reviewer 2: My review changed this part of the content, so these will need to be reintroduced by hand.

Reviewer 2: Fixed in commit.

https://ckeditor.com/docs/cs/latest/onpremises/pdf-onpremises/requirements.html

## Installation

- create imagePullSecret for pulling images from CKEditor container registry,
  replace `xxx` with authentication token
```sh
kubectl create secret docker-registry docker-cke-cs-com \
    --docker-username "pdf" \
    --docker-server "https://docker.cke-cs.com" \
    --docker-password="xxx"
```

- install chart in cluster
>:warning: By default, the chart installs the CKEditor PDF Converter
>on-premises with the "latest" tag. If you are using this chart for a production
>environment, it's strongly recommended to change the container image tag to a
>numeric representation of the version you want to install.
```sh

helm install ckeditor-pdf-converter ./ckeditor-pdf-converter \
    --set image.tag="latest" \
    --set server.secret.data.LICENSE_KEY="" \

>     --set server.ingress.hosts[0].host="test.example"

Reviewer: Same case

```suggestion
    --set ingress.hosts[0].host="test.example" \
    --set ingress.enabled=true
```
